web_src/css/repo.css

```diff
@@ -3083,46 +3083,86 @@ tbody.commit-list {
   }
 }
 
-.pr-status {
+.ui.segment.commit-status-list {
```

Comment:
Can we remove `.ui.segment`? We should not depend on fomantic classes like that. Also I think you can merge some of the rules for `..commit-status-list` like  `transition` into one rule.

Reply:
Done in https://github.com/go-gitea/gitea/pull/26284/commits/3b3f7672196553fc4a90369695d6744352bfa2d0